### ISSUE TITLE
Data Reproduction

### ISSUE THREAD

**Issue by WKGao** [h751410234/DATR#29]
Hi, thanks for sharing this repo.

I have a question：For the cross-domain of C2F, even though I set use_ema=False, the mAP50 in the first stage is 52.0, and the mAP50 in the second stage is 56.02—both are much higher than the 52.8 reported in the paper.Please give me your guidance

**Comment by h751410234** [comment]
Could it be that differences in training hyperparameters, such as the batch size, are causing this issue, since self-training is often quite sensitive to such hyperparameters?

**Comment by h751410234** [comment]
The parameter use_ema has no effect and is essentially an invalid hyperparameter.

**Comment by h751410234** [comment]
If you achieved higher accuracy in the first stage under the same training configuration, I am not entirely sure about the issue and can only assume it is caused by differences in hardware. Empirically, domain adaptation tasks usually exhibit large training fluctuations, and to my knowledge, DINO cannot fix the random seed.
